sky/shell/platform/ios/framework/Source/FlutterViewController.mm

```diff
@@ -420,6 +428,27 @@ - (NSUInteger)supportedInterfaceOrientations {
   return _orientationPreferences;
 }
 
+#pragma mark - Accessibility
+
+- (void)onVoiceOverChanged:(NSNotification*)notification {
+#if TARGET_OS_SIMULATOR
+  // There doesn't appear to be any way to determine whether the accessibility
```

Comment:
Yeah, there's not because there is no VoiceOver on the simulator.  It's replaced by the accessibility inspector, for which there's no API to query (http://stackoverflow.com/questions/35944288/detect-if-the-accessibility-inspector-is-enabled-on-the-simulator)